examples/quad/src/main.rs

```diff
@@ -21,7 +21,7 @@ extern crate gfx_backend_vulkan as back;
 use wasm_bindgen::prelude::*;
 
 #[cfg(target_arch = "wasm32")]
-#[wasm_bindgen(start)]
+#[wasm_bindgen]
```

Comment:
I think we still want `wasm_main` to be the entry point, so `start` should be correct